open-machine-learning-jupyter-book/deep-learning/nn.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>It is recommended to break up the code block into sections that are more readable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>done</p>